scootapi/client/watch_job_cmd.go

```diff
@@ -76,8 +76,12 @@ func PrintJobStatus(jobStatus *scoot.JobStatus) {
 		fmt.Printf("\tTask %s {\n", taskId)
 		fmt.Printf("\t\tStatus: %s\n", taskStatus.String())
 		runStatus := jobStatus.TaskData[taskId]
-		fmt.Printf("\t\tStdout: %v\n", *runStatus.OutUri)
-		fmt.Printf("\t\tStderr: %v\n", *runStatus.ErrUri)
+		if runStatus.OutUri != nil {
```

Comment:
That's not exactly what I'm saying. I'm asserting that my intention in an earlier PR was to guarantee that these were set. If they're not, it probably contradicts some of my code comments.